# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/Thread.cpp
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/Thread.cpp
--- a/gfx/angle/checkout/src/libANGLE/Thread.cpp	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/Thread.cpp	Sat Sep 07 13:38:36 2019 +0000
@@ -11,6 +11,7 @@
 #include "libANGLE/Context.h"
 #include "libANGLE/Debug.h"
 #include "libANGLE/Error.h"
+#include "libANGLE/ErrorStrings.h"
 
 namespace egl
 {
@@ -99,7 +100,7 @@
 {
     if (mContext && mContext->isContextLost())
     {
-        mContext->handleError(GL_OUT_OF_MEMORY, "Context has been lost.", __FILE__, ANGLE_FUNCTION,
+        mContext->handleError(GL_OUT_OF_MEMORY, gl::err::kContextLost, __FILE__, ANGLE_FUNCTION,
                               __LINE__);
         return nullptr;
     }
@@ -107,11 +108,11 @@
     return mContext;
 }
 
-Display *Thread::getCurrentDisplay() const
+Display *Thread::getDisplay() const
 {
     if (mContext)
     {
-        return mContext->getCurrentDisplay();
+        return mContext->getDisplay();
     }
     return nullptr;
 }